### PR TITLE
Coverty fixes for MACs

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: test/evp_test.c [hunk 1]
```diff
@@ -1225,12 +1225,14 @@ static int mac_test_run_mac(EVP_TEST *t)
             goto err;
         }
         tmpval = strchr(tmpkey, ':');
-        if (tmpval != NULL)
+        if (tmpval != NULL) {
```
Comment (slontis) [inline test/evp_test.c]: There is something very funky about the indenting here - where is the matching } for this one??

File: crypto/cmac/cmac.c [hunk 2]
```diff
@@ -146,12 +148,13 @@ int CMAC_Init(CMAC_CTX *ctx, const void *key, size_t keylen,
 int CMAC_Update(CMAC_CTX *ctx, const void *in, size_t dlen)
 {
     const unsigned char *data = in;
-    size_t bl;
+    int bl;
```
Comment (slontis) [inline crypto/cmac/cmac.c]: NIT: extra blankline :)

File: crypto/cmac/cmac.c [hunk 3]
```diff
@@ -89,9 +89,10 @@ int CMAC_CTX_copy(CMAC_CTX *out, const CMAC_CTX *in)
     int bl;
     if (in->nlast_block == -1)
```
Comment (slontis) [inline crypto/cmac/cmac.c]: NIT: blankline

File: test/evp_test.c (hunk 1 above)
Comment (levitte) [reply test/evp_test.c]: Oops, I'll just remove that

File: crypto/cmac/cmac.c (hunk 3 above)
Comment (levitte) [reply crypto/cmac/cmac.c]: Fixed

File: crypto/cmac/cmac.c (hunk 2 above)
Comment (levitte) [reply crypto/cmac/cmac.c]: Fixed

**Comment by slontis** [comment]
travis errors are relevant

**Comment by levitte** [comment]
> travis errors are relevant

I hope the cast I added is enough...

**Comment by levitte** [comment]
> I hope the cast I added is enough...

It looks like it was...

**Review by slontis — APPROVED** [review]
provided that remaining tests pass.

**Comment by levitte** [comment]
Merged

d5f854291336c96a3d2379ecc8c29f00ef516ad9 Coverty fixes for MACs

**Pull request closed without merging**